src/pages/workspace/categories/CategorySettingsPage.tsx

```diff
@@ -138,6 +138,14 @@ function CategorySettingsPage({route, policyCategories, navigation}: CategorySet
                             shouldShowRightIcon
                         />
                     </OfflineWithFeedback>
+                    <OfflineWithFeedback pendingAction={policyCategory.pendingFields?.glCode}>
```

Comment:
Same comment about offline with feedback 🤔  I see that category name has pending fields.... so maybe we would actually want gl code to have it too? probably? maybe need to get confirmation from people in slack, from whoever implemented categories hahaha

Reply:
looks like you already added this, resolving :D